pestion/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "pestion"
```

Comment:
We chatted about this offline, but the idea here was just a portmanteau of Pest and Ion into something that is pronounced like "question".  I was trying to avoid a hyphenated name, so let's get a second opinion from someone like @dlurton / @alancai98 / @abhikuhikar to see if they don't like the name and suggest any alternatives.

If we do like it, I will add something to the README.  If we don't I can refactor the name.

Reply:
Slight preference to `pest-ion` so it doesn't get too closely associated with something David sent offline: https://www.pestions.com/

Reply:
Another possibility is `pest2ion`. It avoids the hyphen and any association with that pest control website.

Reply:
I think the consensus is `pest-ion` so I will refactor it.  The problem with `pest2ion` is that it will look very jarring as a crate (e.g. `use pest2ion`).

Reply:
Changed to `pest-ion`.